filebeat/harvester/forwarder.go

```diff
@@ -21,59 +19,38 @@ type Forwarder struct {
 	Config     ForwarderConfig
 	Outlet     Outlet
 	Processors *processors.Processors
+	meta       common.MapStr
 }
 
 // ForwarderConfig contains all config options shared by all harvesters
 type ForwarderConfig struct {
-	common.EventMetadata `config:",inline"`      // Fields and tags to add to events.
-	Type                 string                  `config:"type"`
-	Pipeline             string                  `config:"pipeline"`
-	Module               string                  `config:"_module_name"`  // hidden option to set the module name
-	Fileset              string                  `config:"_fileset_name"` // hidden option to set the fileset name
-	Processors           processors.PluginConfig `config:"processors"`
+	Type string `config:"type"`
```

Comment:
the forwarder is subject to be removed in future iterations. It's for the PutValue when publishing. In the meantime I added `ClientConfig.Fields` to register fields to be added to every event on publish.
Basically the Forwarder will be remove in favor of the `channel.Outleter`, which will be removed in the future, in favor if `beat.Client` (once we get a more sane registry handling).